<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QTextEdit" name="textEdit"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>640</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Файл</string>
    </property>
    <addaction name="OpenAction"/>
    <addaction name="ExitAction"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>Лог</string>
    </property>
    <addaction name="ExportAction"/>
    <addaction name="AddLogAction"/>
    <addaction name="ViewAction"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="sizeGripEnabled">
    <bool>false</bool>
   </property>
  </widget>
  <action name="ExitAction">
   <property name="text">
    <string>Выход</string>
   </property>
  </action>
  <action name="OpenAction">
   <property name="text">
    <string>Открыть</string>
   </property>
  </action>
  <action name="AddLogAction">
   <property name="text">
    <string>Добавить в лог</string>
   </property>
  </action>
  <action name="ExportAction">
   <property name="text">
    <string>Экспорт...</string>
   </property>
  </action>
  <action name="ViewAction">
   <property name="text">
    <string>Просмотр</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
